lib/generators/blacklight_gallery/install_generator.rb

```diff
@@ -39,5 +39,12 @@ def assets
         "\n//= require blacklight_gallery/blacklight-gallery"
       end
     end
+
+    def uncomment_thumbnail_config
+      gsub_file('app/controllers/catalog_controller.rb',
+                /# config\.index\.thumbnail_field = 'thumbnail_path_ss'/) do |match|
```

Comment:
To future proof this I would use a more forgiving regex:
```suggestion
                /# config\.index\.thumbnail_field.*$'/) do |match|
```